### ISSUE TITLE
Windows 7 support

### ISSUE THREAD

**Issue by BradyBrenot** [BradyBrenot/huestacean#73]
I'm not going to work on this myself. Too many platforms as-is, and I don't have any Windows 7 computers around home to test with. Even my 11-year-old Macbook Pro took the free Windows 10 upgrade just fine, even without Apple's support...

----

It looks like I might have hosed my screen_capture_lite fork's GDI support, or it was hosed upstream, or I'm doing something bad in huestacean that's not letting it work. Getting reports that screen capture on Windows 7 just doesn't work.

**Comment by Collisionc** [comment]
I'll load it up on my windows 7 laptop and run it through debug, if it's something simple/obvious I'll PR it.

**Comment by smasherprog** [comment]
I am the writer of the screen capture library and I used to have support for windows 7 but dropped it since I just dont have any windows 7 machines to test on any more.
The code can brought back into the repo, but I dont have time to maintain it unfortunately.

**Comment by Collisionc** [comment]
So I spent an hour setting up VS2017/qt/etc on the laptop, loaded up the debugger and put some break points in, attached to the debuggable program and found out that the stupid thing works without any issues on windows 7 -- at least on my laptop. 

I Should have just loaded up the release version first :P

**Comment by Collisionc** [comment]
Yeah even when using various settings for displaying to the TV I'm able to sync Win7, so it must be older-hardware related.

**Comment by smasherprog** [comment]
I always thought that it should work, but never had any time to test. Coool

Working on the mac update now. for the library.It should be done today

**Comment by BradyBrenot** [comment]
Cool. I might be a bit delayed pulling that in but when I do I'll let you
know if I have any problems.

**Comment by smasherprog** [comment]
I got most of the mac code in place, but I found areas for code improvement so I am rolling that info this as well. It will take a few more days.. The public api will be the same, but internally, things will be cleaner.
